### DISCUSSION TITLE
Administrators can change the password of the other administrators

### DISCUSSION THREAD

**Discussion by Feiniu6ABC** [minio/minio#13994] [Q&A] [answered]
Sorry for keep asking such obvious questions, but I' still confused on how to disabling administrators from changing the password of the   
other administrators. And still the question, an administrator cannot change his own password even though his password is not set through enviornment variables.

Here is the original question
https://github.com/minio/minio/issues/13987

**Comment by harshavardhana** [comment, marked as answer]
> Sorry for keep asking such obvious questions, but I' still confused on how to disabling administrators from changing the password of the other administrators. And still the question, an administrator cannot change his own password even though his password is not set through enviornment variables.

If you have created users with `consoleAdmin` permissions then you need to add another `Statement` where you need to "Deny"  `admin:CreateUser` - once denied this disallows changing password however they can do all other things on the system.
